src/utils/openid/vci.ts

```diff
@@ -156,8 +164,11 @@ async function exchangePreauthCodeWithToken(endpoint: string, code: string, user
 }
 
 type CredentialOfferMetadata = {
+  credential_definition?: {
+    vct: string;
+  };
   format: string;
-  types: string[];
+  types?: string[]; // TODO: move this types into credential_definition to support jwt_vc_json
```

Comment:
@javereec left this TODO here, will update this when the work on the organisation-wallet is completed